validate.swift

```diff
@@ -422,9 +422,22 @@ do {
     try main(args: CommandLine.arguments)
 } catch {
     if let appError = error as? AppError {
+        if ProcessInfo.processInfo.environment["CI"] == "true" {
+            print("::set-output name=validateError::\(appError.localizedDescription)")
+        }
+
         print("ERROR: \(appError.localizedDescription)")
+        
+        if case .packageListChanged = appError {
+            // For CI it's acceptable for the package list to change as we'll simply take the output of this script
+            exit(EXIT_SUCCESS)
```

Comment:
shouldn’t this be inside the “if CI” block as well?

Reply:
Mentioned in discussion above that it doesn't **need** to be, and practically when someone is running it locally the exit code isn't making a huge amount of difference either.

Not to say it couldn't be moved though

Reply:
Isn’t this changing the shell exit code from success to failure though? My shell actually highlights if a command had a non-zero exit code so it’d be nice to preserve it if it’s easily possible.

Reply:
Ahh my apologies then, I didn't realise folks had it highlighting the exit code.

I'll wrap it into my further tweaks I'm going to make later on 👍  It's a tiny tweak